lib/lamby/proxy_server.rb

```diff
@@ -0,0 +1,36 @@
+module Lamby
+  class ProxyServer
+    
+    METHOD_NOT_ALLOWED = <<-HEREDOC.strip
```

Comment:
Did not use ActiveSupport here intentionally. We never coupled to that since this is a Rack project.